docs/source/index.rst

```diff
@@ -11,6 +11,8 @@ Welcome to Tangelo's documentation!
    :caption: Contents:
 
    overview
+   TUTORIALS
```

Comment:
Do these appear in all caps in our docs?
If yes I think we can simply solve this by renaming them with no caps.
Also, couldn't `docs/source/CONTRIBUTIONS.rst` (resp `TUTORIALS`) simply be symbolic links ?

Reply:
- No it doesn't. It is displaying the content of the file (thus no caps).
- CONTRIBUTIONS.rst and TUTUTIALS.rst are already symbolic links in `docs/source`, if I set it up correctly.